fem/occa.okl

```diff
@@ -44,58 +44,64 @@ typedef double* SymmOperator3D_t @dim(Q3D, 6, NE);
 @kernel void DiffusionSetup2D(const int NE,
                               @restrict const double *W,
                               @restrict const Jacobian2D_t J,
-                              const double COEFF,
-                              @restrict SymmOperator2D_t op) {
-  for (int e = 0; e < NE; ++e; @outer) {
-    for (int q = 0; q < Q2D; ++q; @inner) {
-      const double J11 = J(q, 0, 0, e), J12 = J(q, 1, 0, e);
-      const double J21 = J(q, 0, 1, e), J22 = J(q, 1, 1, e);
-      const double c_detJ = W[q] * COEFF / ((J11 * J22) - (J21 * J12));
-      op(q, 0, e) =  c_detJ * (J21*J21 + J22*J22); // (1,1)
-      op(q, 1, e) = -c_detJ * (J21*J11 + J22*J12); // (1,2), (2,1)
-      op(q, 2, e) =  c_detJ * (J11*J11 + J12*J12); // (2,2)
-    }
-  }
+                              @restrict const DofToQuad_t C,
```

Comment:
I do not know much about OCCA but it looks to me that the dimensions of C are wrong. This is also the case for the 3D setup. Please see the occa.okl file in #1107 for a comparison.

Reply:
You're right, good catch!